ingestion/src/metadata/ingestion/source/dashboard/quicksight/metadata.py

```diff
@@ -120,13 +127,13 @@ def get_dashboard_name(self, dashboard: dict) -> str:
         """
         Get Dashboard Name
         """
-        return dashboard["Name"]
+        return dashboard.Name
 
     def get_dashboard_details(self, dashboard: dict) -> dict:
         """
         Get Dashboard Details
         """
-        return dashboard
+        return DashboardDetail(**dashboard)
 
     def yield_dashboard(
         self, dashboard_details: dict
```

Comment:
can we change the type hint of `dashboard_details` to match the new one in the method signature?

Reply:
Yes , Changed everywhere